core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -110,15 +112,14 @@ oppia.controller('CollectionPlayer', [
       params: {
         stringified_collection_ids: JSON.stringify([$scope.collectionId])
       }
-      }).then(
-        function(response) {
-          $scope.collectionSummary = response.data.summaries[0];
-        },
-        function() {
-          alertsService.addWarning(
-            'There was an error while fetching the collection summary.');
-        }
-      );
+    }).then(
+    function(response) {
```

Comment:
indent by 2, up to line 122; bring the ")" at the end of this block to the next line and deindent to match line 115